### DISCUSSION TITLE
希望后续能增加管理员登录两步验证功能

### DISCUSSION THREAD

**Discussion by isjason** [AlistGo/alist#1445] [Ideas]
希望增加两步验证，可自行选择是否开启，为了更安全的暴露在公网环境，毕竟里面存放了很多个人网盘账号信息

**Comment by orzonzofz** [comment]
这个可以有

**Comment by hsluoyz** [comment]
@xrbzyyyds 已经增加了Casdoor SSO支持：https://alist-doc.nn.ci/docs/setting/casdoor
